src/templates/global/ico_banner.html.tt

```diff
@@ -0,0 +1,16 @@
+<div id="ico_banner" class="ico-banner invisible [% class %]">
+    <div class="gr-row gr-centered">
+        <div class="gr-12">
+            <div class="gr-row gr-no-gutter ico-box">
+                <div class="gr-3 gr-padding-10 ico-column left-bg"></div>
+                <div class="gr-9 gr-padding-10 ico-column">
+                    <div class="gr-row gr-no-gutter">
+                        <div class="gr-9 gr-12-p gr-12-m"><p>[% l('Binary.com ICO starts at 15th November 2017') %]</p></div>
```

Comment:
instead of hardcoding `Binary.com` can you change it to a placeholder? so it would be `l('[_1] ICO starts at 15th November 2017', website_name)`

Reply:
Will do.